utils/cargo-gbuild/src/manifest.rs

```diff
@@ -0,0 +1,46 @@
+// This file is part of Gear.
+//
+// Copyright (C) 2024 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use serde::Deserialize;
+use std::path::PathBuf;
+
+/// Cargo manifest abstraction.
+///
+/// NOTE: considering using `cargo-edit` instead in the future.
```

Comment:
You can use `cargo_toml` crate

Reply:
https://github.com/gear-tech/gear/pull/3835/commits/539214d4bcf29fda941e205f4af611797229cb88